.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: required
+language: python
+services:
+- docker
+env:
+  global:
+  - IMAGE=praekeltfoundation/logspout-sumologic
+  - REGISTRY_USER=praekeltorgdeploy
+  - secure: pEOV9lQoxvCCWrDn+XiCkhMTPULJ9G2cFPr229n6YA6v8XLeC8jsfGJMmXtgdczPK79q6nnfZEr1EGZDcAK2dbkHGowkIXo096io65tCjsydUn2rGyajizePWXz4ycENIKeWcAxA7eUVhxRoRVY5Q4aTCHM+ZGzalBqdxFKdBDOEvok3FuDyLnwUZjVuM748295FGHbE/lz30NORfgAIkuWOEG16PlaIYutcnOWYHVLmvwfEkkZ/77eCWwymafZ2RNYn/l4WNr5W6mC4OqCDb90+tQ/o+VJyhJzy8FNzbZUecs7PXKKr8uzZhS6+L6KgwxLXRS4WOOxvTAKV3u9x7ByH8prvLWdhxuhfSSRcDFVGmrMWry/5WcSBmBtqORcbTo0ahv6eoR+C8IHZbZ2Hz5Oi0Y+OLFwXOcxF4wPabxJkQyeXyM+RgQ19l69TCi086ez3KHP38tUEr1owbAVYTM9BhjAQf5XBZchwNQqpyXzJsiueDEoelBSDfnSaqWR+F2SYXB/liuUBw9Z9W8nTNgjFb3GIyv4rxzsHPLn94sswc09tPsPSNabxVC+0/pHqXMTpzQ+NsyffXIjYifda+qWugCbLP60Vp0WWr+cE7TZwm42LOjmKOCV0ATtQNityfmxvdNLeSlX1nBxfv3mbJTybvZI8bQF1uOb456cqcI8=
+before_install:
+- sudo apt-get update
+- sudo apt-get install -qy -o Dpkg::Options::="--force-confold" docker-ce
+install: []
+before_script:
+- cd docker
+- version="$(awk '$2 == "LOGSPOUT_SUMOLOGIC_VERSION" { print $3; exit }' Dockerfile | cut -d'~'
+  -f1)"
+- docker pull "$IMAGE" || true
+script:
+- docker build --pull --cache-from "$IMAGE" -t "$IMAGE" .
+before_deploy:
+- pip install docker-ci-deploy==0.2.0
```

Comment:
Latest is 0.3.0 which changes the option names (although I think the old options still work). Command becomes:
```
dcd --version "$version" --version-semver --version-latest "$IMAGE"
```